Cargo.toml

```diff
@@ -21,3 +21,7 @@ members = ["."]
 
 [profile.release]
 lto = true
+strip = "symbols"
+panic = "abort"
```

Comment:
`panic = "abort"` is the only available option for the wasm target. Not sure if we need to specify it here.